distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFederation.java

```diff
@@ -89,6 +112,8 @@
 
     private static final String DEFAULT_SAMPLE_PRODUCT_FILE_NAME = "sample.txt";
 
+    private static final DynamicPort RESTITO_STUB_SERVER_PORT = new DynamicPort(41300);
```

Comment:
this is not as cool as I thought it was the way it currently is setup you have to make sure noone else is using your dynamic port....